### PR TITLE
[s]Autocloning code improvements

### PR DESCRIPTION
**Pull request by silicons**
Moves minimal stock part level to a define for easy access
Fixes href exploit that allows enabling autocloning without enough upgrade level.

### CODE REVIEW

File: code/game/machinery/computer/cloning.dm [hunk 1]
```diff
@@ -151,7 +153,7 @@
 	var/dat = ""
 	dat += "<a href='byond://?src=[REF(src)];refresh=1'>Refresh</a>"
 
-	if(scanner && HasEfficientPod() && scanner.scan_level > 2)
+	if(scanner && HasEfficientPod() && scanner.scan_level >= AUTOCLONING_MINIMAL_LEVEL)
 		if(!autoprocess)
 			dat += "<a href='byond://?src=[REF(src)];task=autoprocess'>Autoprocess</a>"
```
Comment (steamp0rt) [inline code/game/machinery/computer/cloning.dm]: 😋 time to post how to do this in ooc

**Comment by vuonojenmustaturska** [comment]
this is not what [s] is for, also include a changelog for the balance changes you did

**Comment by Thunder12345** [comment]
But... there's no balance changes here?

**Comment by kevinz000** [comment]
image

**Comment by vuonojenmustaturska** [comment]
oh yeah nvm i misread the code as you said

**Pull request merged**